### ISSUE TITLE
add a REST API we can invoke from `gofabric8` to update a tenant passing just the OC token

### ISSUE THREAD

**Issue by jstrachan** [fabric8-services/fabric8-tenant#222]
we'd need to then do a reverse lookup of the KC token inside the REST endpoint so that we can do an update tenant.

this would then mean we could have a `gofabric8 recreate tenant` type command which deletes a tenant, invokes the update and then asserts the tenant is setup correctly etc.

It might be nice to allow a version to be passed in as an optional argument too.

**Comment by chmouel** [comment]
I don't think we have a way to be able to reverse a OC token to a KC token without authing to KC, we may need to do ACL lists instead like done for `tenant_listing` here : 

https://github.com/fabric8-services/fabric8-tenant/pull/371/commits/a18fbc3619cce46bc6230ac732e1a6f10767ef23
